flow/util/genRuleFile.py

```diff
@@ -130,25 +130,25 @@
         'roundValue': True,
         'compare': '<=',
     },
-    'finish__timing__drv__max_slew': {
+    'finish__timing__drv__max_slew_limit': {
         'usePeriod': False,
         'padding': 20,
-        'fixed': 10,
-        'roundValue': True,
+        'absolute': True,
+        'roundValue': False,
```

Comment:
Would you clarify exactly what the final check is?  We've had a lot of discussions/permutations.

Reply:
```
metric_rule = abs(sta::max_slew_check_slack_limit) + 20%
```
where `max_slew_check_slack_limit` is:
```
[sta::max_fanout_check_slack] / [sta::max_fanout_check_limit]
```
So is the slack of the worst violation (slew, cap, fanout) divided by their respective limit. The `20%` is the `padding`, i.e, the margin we allow the metric to change for the worst.

Reply:
Commit [e6d3a60](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/pull/424/commits/e6d3a600493c7557c74bcb3c387a3d64e4c1c5bb) implements the changes discussed on slack.